private/pkg/git/git.go

```diff
@@ -173,276 +135,3 @@ type ListFilesAndUnstagedFilesOptions struct {
 	// is being applied to.
 	IgnorePathRegexps []*regexp.Regexp
 }
-
-// Hash represents the hash of a Git object (tree, blob, or commit).
-type Hash interface {
```

Comment:
There's one place in core that uses `Hash`, we should see if we can remove it?

Reply:
You're right, here a PR that removes it: https://github.com/bufbuild/core/pull/13382